neanderthal/src/main/kotlin/au/com/outware/neanderthal/presentation/view/activity/VariantListActivity.kt

```diff
@@ -36,7 +36,7 @@ class VariantListActivity : AppCompatActivity(), VariantListPresenter.ViewSurfac
         adapter = VariantAdapter { name, position -> presenter.onItemSelected(name, position) }
 
         presenter = VariantListPresenter(this, adapter)
-        presenter.onCreate(savedInstanceState)
+        presenter.onCreate(null)
```

Comment:
Fair question. updated the pattern a bit to throw in extras + saved instance state